lessonbuilder/tool/src/webapp/templates/ShowPage.html

```diff
@@ -887,8 +887,8 @@ <h3 class="questionText" rsf:id="questionText"></h3>
                                     <form action="#" rsf:id="shortanswerForm" class="shortanswerForm">
 				      <input type="hidden" rsf:id="csrf5" />
                                         <input type="hidden" rsf:id="shortanswerId" />
-                                        <div rsf:id="msg=simplepage.answer"></div>
-                                        <textarea rsf:id="shortanswerInput" class="resizable" ></textarea>
+                                        <div for="answerInput" rsf:id="msg=simplepage.answer"></div>
+                                        <textarea rsf:id="shortanswerInput" aria-labelledby="answerInput"></textarea>
```

Comment:
I don't think this is going to work. for="answerInput" should be id="answerInput" I think.

Reply:
Ok, this should work.